### PR TITLE
Ignore reviews that have not yet been submitted

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

## Description

Allow submitted at to be misisng from a review - which occurs if a review is still in process.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by ghalliday** [comment]
@richardkchapman please review

**Comment by richardkchapman** [comment]
Is this desirable? You'll end up with ownership transferred to the person mentioned in the (unsubmitted) review, but they won't be able to see the review that they are expected to respond to. I would have thought you want ownership to stay with the person that is mid-review

**Comment by ghalliday** [comment]
@richardkchapman version 2...

File: pulls.py [hunk 1]
```diff
@@ -156,6 +156,9 @@ def _fetch_reviews(self, github_repo, pull):
             for review in reviews:
                 body = review['body']
                 mentions = re.findall(r"@([-A-Za-z0-9]+)", body)
+                #If the review hasn't been submitted then do not process it
```
Comment (richardkchapman) [inline pulls.py:159]: Minor quibble: would be better a couple of lines higher

**Pull request merged**